.github/workflows/build-net6.0.yml

```diff
@@ -21,28 +21,32 @@ jobs:
     strategy:
       matrix:
         os: [ ubuntu-latest, windows-latest ]
-        dotnet-version: [ '6.0.x' ]
 
     steps:
       - uses: actions/checkout@v4
       
       - name: Setup .NET
         uses: actions/setup-dotnet@v4
         with:
-          dotnet-version: ${{ matrix.dotnet-version }}
+          dotnet-version: 6.0.x
```

Comment:
is it a good idea to use `6.0.x`? isn't better/safer to lock a specific version?

Reply:
Let's keep the .NET version up-to-date with the latest security-patches: https://dotnet.microsoft.com/en-us/download/dotnet/8.0